lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/KeyStoreService.java

```diff
@@ -17,21 +14,16 @@
 
 public class KeyStoreService {
     private static final Logger LOGGER = LoggerFactory.getLogger(KeyStoreService.class);
-    private final SecretsProvider secretsProvider;
-    private final String secretKeyPrefix;
+    private AWSSecretsRetriever awsSecretsRetriever;
 
     public KeyStoreService() {
-        this(
-                ParamManager.getSecretsProvider(),
-                System.getenv(ConfigurationConstants.AWS_STACK_NAME));
+        this(new AWSSecretsRetriever());
```

Comment:
Use the ConfigurationService in this class to obtain secret values from AWS
Add a new method to ConfigurationService to get the secret by a specified key